state-chain/pallets/cf-ingress-egress/src/lib.rs

```diff
@@ -566,12 +590,15 @@ pub mod pallet {
 	impl<T: Config<I>, I: 'static> Hooks<BlockNumberFor<T>> for Pallet<T, I> {
 		/// Recycle addresses if we can
 		fn on_idle(_n: BlockNumberFor<T>, remaining_weight: Weight) -> Weight {
-			let read_write_weight =
-				frame_support::weights::constants::RocksDbWeight::get().reads_writes(1, 1);
+			let mut used_weight = Weight::zero();
+
+			let cleanup_weight = frame_support::weights::constants::RocksDbWeight::get()
```

Comment:
Should this be `recycle_weigth_per_block`? Also, maybe we could comments where `reads_writes(2, 2)` comes from?